packages/ui/src/max-width-wrapper.tsx

```diff
@@ -10,10 +10,7 @@ export function MaxWidthWrapper({
 }) {
   return (
     <div
-      className={cn(
-        "mx-auto w-full max-w-screen-xl px-2.5 lg:px-20",
-        className,
-      )}
+      className={cn("mx-auto w-full max-w-screen-xl px-3 lg:px-10", className)}
```

Comment:
@steven-tey I think this `MaxWidthWrapper` component is used in the marketing site so we'll need to double check / tweak things on that end.

Reply:
ooooh yes 👀 I can publish a new version for `@dub/ui` to check